tests/codegen/simd/simd-wide-sum.rs

```diff
@@ -52,8 +52,6 @@ pub fn wider_reduce_iter(x: Simd<u8, N>) -> u16 {
 #[no_mangle]
 // CHECK-LABEL: @wider_reduce_into_iter
 pub fn wider_reduce_into_iter(x: Simd<u8, N>) -> u16 {
-    // CHECK: zext <8 x i8>
-    // CHECK-SAME: to <8 x i16>
-    // CHECK: call i16 @llvm.vector.reduce.add.v8i16(<8 x i16>
+    // CHECK: call i16 @llvm.vector.reduce.add
```

Comment:
```suggestion
    // FIXME: It would be nice if this was exactly the same as the above tests,
    // but at the time of writing this comment it wasn't on LLVM main.
    // CHECK: call i16 @llvm.vector.reduce.add
```

Reply:
Thanks. I believe it should be the same when this test case starts with LLVM 17.

Reply:
> I believe it should be the same when this test case starts with LLVM 17.

Glad to hear it!  I'm not worried about it checking less -- it's fine if it's that way for even a long time -- just wanted to make sure there was a clear "why is this difference" for people to see.

(Sorry for taking a while to get back to reviewing this 😬)